### ISSUE TITLE
new foundation for apps install failed npm error log

### ISSUE THREAD

**Issue by egad1868** [npm/npm#7024]
![npm](https://cloud.githubusercontent.com/assets/6023583/5562441/2f2049bc-8deb-11e4-8d0c-d287d0daac7e.PNG)

**Comment by egad1868** [comment]
I got it working! Not sure how exactly or what finally did it, but it's working! 

I did use a couple of other tutorials that I am sure helped one part or another. Hopefully they may help someone else:

BIGGEST thing I learned through this whole process: If your most recent action doesn't seem to have worked  try this FIRST: You MUST close your command prompt and re open it when checking if installs and commands worked or not! 

~Had to add Ruby bin to my PATH manually this recommends re-install and choose "Add Ruby executables to your PATH" but doing so manually was extremely easy. (and is NOT specific to Wordpress which I don't use)

http://awordpress.net/blog/ruby-is-not-recognized-as-an-internal-or-external-command-operable-or-batch-file-error/

~I temporarily gave up on foundation for apps and just went with a sass/compass install and when I got that working I tried foundation again (a few days later) and was successful this site helped me with the compass/sass install 

http://thesassway.com/beginner/getting-started-with-sass-and-compass

googling my specific errors led me to things that needed fixing though I had to do a little digging to find the right answers.

Good Luck to those that follow hopefully your journey won't be as frustrating.
